Makefile

```diff
@@ -14,7 +14,7 @@ lint:
 test-start:
 	# use a non-standard port so that it doesn't collide with the dev
 	# environment
-	docker run -d --name $(RABBITMQ_CONTAINER) \
+	docker run --env RABBITMQ_DEFAULT_VHOST=queue_vhost -d --name $(RABBITMQ_CONTAINER) \
```

Comment:
@msuvam Can you change the name of the PR to reflect what it is about? It is not just queue package update

Reply:
ok sure

Reply:
@krisowski   i have updated the name of the PR, please suggest if any more changes